test/plugin/base_test.rb

```diff
@@ -274,6 +274,13 @@ def test_invalid_json_credentials
     end
   end
 
+  def test_unset_empty_credential_env_var
+    ENV[CREDENTIAL_PATH_VAR] = ''
```

Comment:
If you decide to keep the constant, should we change all uses of `'GOOGLE_APPLICATION_CREDENTIALS'` in this file to use this constant?

Reply:
Yup. Done.